src/views/linkView.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Create an Link element
+ * @returns {Element}
+ */
+export const createLinkElement = (href, label) => {
+  const element = document.createElement('div');
```

Comment:
Why do you need a div here?

Reply:
because there can be 1, 2, or more links and they don't stick nicely so it is better to have the possibility to style them later